x/marker/keeper/marker.go

```diff
@@ -697,6 +700,62 @@ func (k Keeper) TransferCoin(ctx sdk.Context, from, to, admin sdk.AccAddress, am
 	return nil
 }
 
+// IbcTransferCoin transfers restricted coins between to chains when the administrator account holds the transfer
+// access right and the marker type is restricted_coin
+func (k Keeper) IbcTransferCoin(
+	ctx sdk.Context,
+	sourcePort, sourceChannel string,
+	token sdk.Coin,
+	sender, admin sdk.AccAddress,
+	receiver string,
+	timeoutHeight clienttypes.Height,
+	timeoutTimestamp uint64,
+	checkRestrictionsHandler ibckeeper.CheckRestrictionsHandler) error {
+	m, err := k.GetMarkerByDenom(ctx, token.Denom)
+	if err != nil {
+		return fmt.Errorf("marker not found for %s: %w", token.Denom, err)
+	}
+	if m.GetMarkerType() != types.MarkerType_RestrictedCoin {
+		return fmt.Errorf("marker type is not restricted_coin, brokered transfer not supported")
+	}
+	if !m.AddressHasAccess(admin, types.Access_Transfer) {
+		return fmt.Errorf("%s is not allowed to broker transfers", admin.String())
+	}
+	if !admin.Equals(sender) {
```

Comment:
The admin right is not the correct grant to use for a transfer operation.  That is what the transfer grant is for.